protocol/testutil/ante/testutil.go

```diff
@@ -109,7 +122,7 @@ func (suite *AnteTestSuite) CreateTestAccounts(numAccs int) []TestAccount {
 	for i := 0; i < numAccs; i++ {
 		priv, _, addr := testdata.KeyTestPubAddr()
 		acc := suite.AccountKeeper.NewAccountWithAddress(suite.Ctx, addr)
-		if err := acc.SetAccountNumber(uint64(i)); err != nil {
+		if err := acc.SetAccountNumber(uint64(i + 1000)); err != nil {
```

Comment:
Any reason for +1000?

Reply:
It was a holdover from me debugging something. I swapped it back.